scripts/training/train.py

```diff
@@ -666,6 +667,7 @@ def main(
         torch_compile=torch_compile,
         ddp_find_unused_parameters=False,
         remove_unused_columns=False,
+        use_cpu=True,
```

Comment:
What does this to?

Reply:
Uhm, I'm not sure how this change got here honestly